stable/blackbox-exporter/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if gt (len .Values.config) 0 }}
```

Comment:
I'd just do this instead:

`{{- if .Values.config }}`

Reply:
ok.